newIDE/app/src/ObjectEditor/Editors/SpriteEditor/index.js

```diff
@@ -75,28 +77,37 @@ class Animation extends React.Component<AnimationProps, void> {
       onReplaceDirection,
       objectName,
       onChangeName,
+      isAnimationListLocked,
     } = this.props;
 
     const animationName = animation.getName();
     return (
       <div>
-        <MiniToolbar noPadding>
-          <DragHandle />
-          <MiniToolbarText>Animation #{id} </MiniToolbarText>
+        {isAnimationListLocked && (
           <Column expand>
-            <SemiControlledTextField
-              commitOnBlur
-              margin="none"
-              value={animation.getName()}
-              translatableHintText={t`Optional animation name`}
-              onChange={text => onChangeName(text)}
-              fullWidth
-            />
+            <Spacer />
```

Comment:
Could we do without this spacer?

Reply:
(Column already has some spacing on the left/right)

Reply:
It's for the top, the title was right under the previous animation sprite.

Reply:
Ok, check if a Line can do, otherwise it's ok

Reply:
A Line works